public/robots.txt

```diff
@@ -2,6 +2,8 @@
 #
 # To ban all spiders from the entire site uncomment the next two lines:
  User-agent: *
- Disallow: /flags
+ Disallow: /admins
+ Disallow: /blockeds
+ Disallow: /activities
 
  Sitemap: http://www.catforum.mntc.in/sitemap.xml
```

Comment:
> isn't blocked, activities and admins covered with **authorization** as well?

I am sorry, I mean **authentication**. And I am pretty sure you need to authenticated as admin user to access those pages.